src/HH4b/boosted/TrainBDT.py

```diff
@@ -1059,19 +1220,28 @@ def main(args):
         type=str,
         nargs="+",
     )
+    parser.add_argument(
```

Comment:
Not a big deal, but do we need the `--xbb` and `--mass` options if we have the `--legacy` option?